components/search/SearchQueryOptions.vue

```diff
@@ -0,0 +1,426 @@
+<template>
+  <b-list-group
+    :id="elementId"
+    class="auto-suggest-dropdown"
+    data-qa="search suggestions"
+    role="listbox"
+    :aria-label="$t('searchSuggestions')"
+    :aria-hidden="!isActive"
+  >
+    <template
+      v-if="onCollectionPage"
+    >
+      <b-list-group-item
+        :to="inCollectionLinkGen(query)"
+        class="search"
+        data-qa="search in collection button"
+        role="option"
+        :aria-label="$t('search')"
+        :aria-selected="focus === 0"
+        :class="{ 'hover': focus === 0 }"
+        :data-index="0"
```

Comment:
I am not sure if the data-index still has a purpose, especially because not all list-group-items have them.